### DISCUSSION TITLE
Plugin to support other numerical formats

### DISCUSSION THREAD

**Discussion by Miki-AG** [dream-num/univer#2009] [Q&A | 常见问题]
Is there an example of a plugin that shows how to transform the displayed value of the cells? I want to write a simple plugin that enables a different numerical formatting than the ones provided by the app by default.

**Comment by Gggpound** [comment]
To customize a number format rule, consider using the commandService to call set-numfmt.command (located in packages/sheets-numfmt/src/commands/commands/set-numfmt.command.ts). For more information on how to use the commandService, please refer to this chapter: https://univer.ai/guides/tutorials/csv-import-plugin/.
